### PR TITLE
[PFX-743] - Quick Start Doc

### PR DESCRIPTION
**Pull request by mmason2-godaddy**
## Summary
- Quick Start Doc

## Changelog
- Quick Start Doc

## Test Plan
n/a

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
 
 ```bash
 npm i -g create-gasket-app
+# OR
+npx create-gasket-app
 ```
 
-Now you can create gasket apps with the `create-gasket-app` command and the preset
-you would like to use. In this example we will create a Next.js app.
+Now you can create gasket apps with the `create-gasket-app` command, the `appname` and the preset you would like to use. In this example we will create a Next.js app. The `appname` is the only required argument.
```
Comment (agerard-godaddy) [inline docs/quick-start.md:13]:
```suggestion
In this example we will create a Next.js app by specifying the app name along with the preset `@gasket/preset-nextjs`:
```

File: docs/quick-start.md [hunk 2]
```diff
@@ -33,7 +34,7 @@ Otherwise you can build and start your app directly:
 
 ```bash
 npm run build
-npm run start --env local
+GASKET_ENV=local npm run start
```
Comment (agerard-godaddy) [inline docs/quick-start.md:37]:
```suggestion
npm run local
```

Comment (mmason2-godaddy) [reply docs/quick-start.md:37]: I'm not sure what the change is here. I have the local command in the section above. Do you want me to remove the entire seciton?

Comment (agerard-godaddy) [reply docs/quick-start.md:37]: My bad... 

So, to be concise, leave out the GASKET_ENV in the example.
```
npm run build
npm run start
```

local is the default env now

**Review by agerard-godaddy — APPROVED** [review]

**Review by jpina1-godaddy — APPROVED** [review]

**Pull request merged**